shanoir-ng-datasets/src/main/java/org/shanoir/ng/download/WADODownloaderService.java

```diff
@@ -315,9 +317,17 @@ public List<File> downloadDicomFilesForURLs(final List<URL> urls, final File wor
 
 	public String downloadDicomMetadataForURL(final URL url) throws IOException, MessagingException, RestClientException {
```

Comment:
Hi @julien-louis, I think both methods downloadDicomMetadataForURL should be fusioned please. With kind regards, Michael